lineage/figures/figure2.py

```diff
@@ -66,7 +66,7 @@ def accuracy():
         list_of_fT.append(T)
         list_of_fE.append(E2)
 
-    total_cellnum, paramEst, accuracy_after_switching, transition_matrix_norm, pi_vector_norm, paramTrues = commonAnalyze(list_of_populations)
+    total_cellnum, paramEst, accuracy_after_switching, transition_matrix_norm, pi_vector_norm, paramTrues, _ = commonAnalyze(list_of_populations)
```

Comment:
It might just be better to make the `accuracy_after_switching` the `balanced_accuracy_score` in general